src/server/routes/sign.rs

```diff
@@ -81,12 +81,9 @@ impl SignRequest {
             error!("Failed to build data for signing: {:?}", err);
         })?;
 
-        let decoded_signature =
-            BASE64URL_NOPAD
-                .decode(self.signature.as_bytes())
-                .map_err(|err| {
-                    error!("Failed to decode signature: {:?}", err);
-                })?;
+        let decoded_signature = BASE64.decode(self.signature.as_bytes()).map_err(|err| {
```

Comment:
Is this right? Earlier @joshbenaron said this should be `BASE64URL_NOPAD`.

Reply:
Testing with bundlr-node sending signatures, it does not decode/encode correctly with `BASE64URL_NOPAD`

Reply:
After some trial and error, I was able to get it with just `BASE64`

Reply:
In https://docs.arweave.org/developers/server/http-api#transaction-format, it's defined to be `BASE64URL`. Maybe it's just not `BASE64URL_NOPAD`

Reply:
Alright, 2 PRS then

Reply:
Has the second PR been created?